GitExtensions.sln

```diff
@@ -150,6 +150,29 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "JiraCommitHintPlugin", "Plu
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonTestUtils", "UnitTests\CommonTestUtils\CommonTestUtils.csproj", "{7B1F2FA0-E17D-499B-AF86-D49DBEF8C548}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{C5855184-8571-4AD0-A373-B98215BA56CA}"
+	ProjectSection(SolutionItems) = preProject
+		.codecov.yml = .codecov.yml
+		.editorconfig = .editorconfig
+		.gitattributes = .gitattributes
+		.gitignore = .gitignore
+		.gitmodules = .gitmodules
+		.mailmap = .mailmap
+		.travis.yml = .travis.yml
+		appveyor.yml = appveyor.yml
+		CommonAssemblyInfo.cs = CommonAssemblyInfo.cs
+		CONTRIBUTING.md = CONTRIBUTING.md
+		CustomDictionary.xml = CustomDictionary.xml
+		Directory.Build.props = Directory.Build.props
+		FxCop.targets = FxCop.targets
+		FxCopRules.ruleset = FxCopRules.ruleset
+		GitExtensions.ruleset = GitExtensions.ruleset
+		GitExtensionsTest.ruleset = GitExtensionsTest.ruleset
+		LICENSE.md = LICENSE.md
+		README.md = README.md
+		stylecop.json = stylecop.json
+	EndProjectSection
+EndProject
```

Comment:
Added a bunch of top-level files to the solution to allow editing them from within VS.